src/planscape/forsys/get_forsys_inputs.py

```diff
@@ -83,6 +89,11 @@ def _read_url_params_with_defaults(self, params: QueryDict) -> None:
         else:
             self.project_areas = self._get_default_project_areas()
 
+        self.max_area_in_km2 = self._read_positive_float(params,
```

Comment:
It already does this: if the field is missing, _read_positive_float returns None.
I'll clarify in a comment.

Reply:
oh yes i saw that! sorry i meant for this method in general (region and priorities in particular)

Reply:
sorry, github doesn't allow me to attach comments to parts of the code that aren't changed

Reply:
I see! I did that on purpose so that I can debug/sanity check backend logic on real data without bringing up a frontend.

In production, the common path will involve reading these parameters from a db.